src/content/learn/react-developer-tools.md

```diff
@@ -4,30 +4,31 @@ title: React Developer Tools
 
 <Intro>
 
-Use React Developer Tools to inspect React [components](/learn/your-first-component), edit [props](/learn/passing-props-to-a-component) and [state](/learn/state-a-components-memory), and identify performance problems.
+Gunakan React Developer Tools untuk memeriksa [komponen](/learn/your-first-component), menyunting [*props*](/learn/passing-props-to-a-component) dan [*state*](/learn/state-a-components-memory), dan mengidentifikasi masalah kinerja.
 
 </Intro>
 
 <YouWillLearn>
 
-* How to install React Developer Tools
+* Cara menginstal React Developer Tools
```

Comment:
Cara memasang

Reply:
According to the [Glossary](https://github.com/reactjs/id.react.dev/wiki/Glosarium#i), it's fine to use `menginstal`

Reply:
Yep, let's keep it `instal`

Reply:
@resir014 @mhaidarhanif Nice, need to check this one also, belum menggunakan `instal` https://github.com/reactjs/id.react.dev/pull/333/files